instrumentation/instabeego/web_server_handler_test.go

```diff
@@ -64,31 +63,31 @@ func initBeeApp(t *testing.T, r *instana.Recorder) {
 	sleep(serverDepTime)
 }
 
-func TestMain(m *testing.M) {
-
-	instana.InitSensor(&instana.Options{
+func initSensor() {
```

Comment:
Can we change this to `initCollector` to remove the ambiguity ?

Reply:
yes 👍